src/query/functions/src/scalars/bitmap.rs

```diff
@@ -143,6 +144,36 @@ pub fn register(registry: &mut FunctionRegistry) {
         ),
     );
 
+    registry.register_passthrough_nullable_3_arg::<BitmapType, UInt64Type, UInt64Type, BitmapType, _, _>(
+        "sub_bitmap",
+        |_, _, _| FunctionDomain::MayThrow,
+        vectorize_with_builder_3_arg::<BitmapType, UInt64Type, UInt64Type, BitmapType>(
+            |b, offset, length, builder, ctx| {
+                    match RoaringTreemap::deserialize_from(b) {
+                        Ok(rb) => {
+                            let subset_start = offset;
+                            let subset_length = length;
+                            if subset_start >= b.len() as u64 {
```

Comment:
If the offset exceeds the length of the bitmap, we can return an empty bitmap without reporting an error.

Reply:
Done